Solutions/TransmitSecurity/Data/Solution_TransmitSecurity.json

```diff
@@ -8,7 +8,7 @@
       "Data Connectors/TransmitSecurity_API_FunctionApp.json"
     ],
     "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\TransmitSecurity\\",
-    "Version": "3.0.1",
+    "Version": "3.0.2",
```

Comment:
@v-prasadboke
I have a question about this. I changed the version value to 3.0.2, but the solution that was created is named 3.0.1.zip. Is that expected? Is it okay?

Reply:
Ok got it. Thanks